rust-runtime/aws-smithy-http-server/src/body.rs

```diff
@@ -10,9 +10,34 @@ pub use http_body::Body as HttpBody;
 #[doc(hidden)]
 pub use hyper::body::Body;
 
+use bytes::Bytes;
+
+use crate::error::{BoxError, Error};
+
+pub type BoxBody = http_body::combinators::UnsyncBoxBody<Bytes, Error>;
```

Comment:
This type should be `#[doc(hidden)]` because it is not meant to be directly used by the user, but only by the code generator. We use `#[doc(hidden)]` to signal that is is public API that is not supported outside of the codegenerator.